.env.example

```diff
@@ -1,4 +1,4 @@
-DEFI_ADAPTERS_LOG_LEVEL=debug
+DEFI_ADAPTERS_LOG_LEVEL=info
```

Comment:
Default log level moved to info because using debug, when running new adapter command, it'll ask the first question and then dump all sort of debug lines in between.